app/src/behaviors/behavior_transparent.c

```diff
@@ -9,26 +9,28 @@
 #include <zephyr/device.h>
 #include <drivers/behavior.h>
 #include <zephyr/logging/log.h>
+#include <zmk/keymap.h>
 
 #include <zmk/behavior.h>
 
 LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
 
 #if DT_HAS_COMPAT_STATUS_OKAY(DT_DRV_COMPAT)
 
-static int on_keymap_binding_pressed(struct zmk_behavior_binding *binding,
+static int on_keymap_binding_trigger(struct zmk_behavior_binding *binding,
                                      struct zmk_behavior_binding_event event) {
-    return ZMK_BEHAVIOR_TRANSPARENT;
-}
-
-static int on_keymap_binding_released(struct zmk_behavior_binding *binding,
-                                      struct zmk_behavior_binding_event event) {
-    return ZMK_BEHAVIOR_TRANSPARENT;
+    // Avoid uint8_t overflow resulting in an infinite loop
+    if (LAYER_ID_TO_INDEX(event.layer) == 0) {
```

Comment:
Store this to a local if we're gonna be using this twice, since it may involve some lookups.